src/System.CommandLine/Symbol.cs

```diff
@@ -45,7 +46,21 @@ public abstract class Symbol : ISymbol
 
         public IReadOnlyCollection<string> RawAliases => _rawAliases;
 
-        public Argument Argument { get; set; }
+        public Argument Argument 
+        { 
+            get
+            {
+                return _argument;
+            }
+            set
+            {
```

Comment:
One last thing I realized I missed previously. We have the assumption that the `Argument` will never be null. I believe we should check for `null` and in that case use `Argument.None` like what is done in the ctor.